slartibartfast/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link href='https://fonts.googleapis.com/css?family=Alex Brush' rel='stylesheet'>
+    <link href='https://fonts.googleapis.com/css?family=Allura' rel='stylesheet'>
+    <title>Glimåkra Webbutvecklare Restaurangmenyer</title>
+
+</head>
+
+<body>
+
+    <header>
+        <section id="huvud" class="lös">
+            <img src="../lunchtimeweb-01_f73305aa-cd47-495d-9457-7a53dfcb022e_1024x1024@2x.png" alt="" width="200px">
+        <h1>Slartibartfast Kulinarer</h1>
+        <p></p>
+    </header>
+    <main>
+        <ul>
+            <li>
+                <a href="menyer/exempel/index.html">Menyer </a>
+            <li>
+                <h1>Meny</h1>
+            </li>
+            </li>
```

Comment:
Du stänger båda li taggarna i slutet av ul taggen.
```suggestion
            <li>
                <a href="menyer/exempel/index.html">Menyer </a>
            </li>
            <li>
                <h1>Meny</h1>
            </li>
```